packages/wonder-blocks-form/src/components/text-area.tsx

```diff
@@ -388,7 +388,7 @@ const styles = StyleSheet.create({
         },
         ":focus-visible": {
             // TODO(WB-1856): Verify if we can use the global focus color
```

Comment:
Can this TODO comment be removed with this PR, or is that different work?

Reply:
This is happening here: https://github.com/Khan/wonder-blocks/pull/2480